source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -93,7 +93,25 @@ edX project. There are many Django microservices in the Open edX ecosystem, such
 #. If you are creating a new repository, use the `cookiecutter-django-ida`_
    template.
 
-**TODO:** How to enable translations infrastructure on a Django microservice
+#. For existing repos which don't have the ``make extract_translations`` or
+   ``make pull_translations`` command, they can be copied from the
+   `edx-cookiecutters`_ Makefile in the ``cookiecutter-django-ida`` Makefile for Microservices.
```

Comment:
Having these as numbered steps in a row makes this somewhat confusing. It currently renders as
1. If you are creating a new repository
2. For existing repos

It seems to me they should both be part of step 1.